spyder/plugins/application/container.py

```diff
@@ -302,31 +302,40 @@ def _check_updates_ready(self):
                 header = _("<b>Spyder {} is available!</b> "
                            "<i>(you have {})</i><br><br>").format(
                     latest_release, __version__)
-                content = ""
+                download_msg = _(
+                    "Would you like to automatically download and "
+                    "install it?<br><br>"
+                )
+                pip_warn_msg = _(
+                    "<b>Important note:</b> Since you installed "
+                    "Spyder with Anaconda,{} please <b>don't</b> use "
+                    "<code>pip</code> to update it as that will break "
+                    "your installation.<br><br>"
+                    "Instead, run the following commands in a "
+                    "terminal:<br>"
+                    "<code>conda update anaconda</code><br>"
+                    "<code>conda install spyder={}</code><br><br>"
+                )
                 footer = _(
                     "For more information, visit our "
                     "<a href=\"{}\">installation guide</a>."
                 ).format(url_i)
-                if is_anaconda():
-                    content = _(
-                        "<b>Important note:</b> Since you installed "
-                        "Spyder with Anaconda, please <b>don't</b> use "
-                        "<code>pip</code> to update it as that will break "
-                        "your installation.<br><br>"
-                        "Instead, run the following commands in a "
-                        "terminal:<br>"
-                        "<code>conda update anaconda</code><br>"
-                        "<code>conda install spyder={}</code><br><br>"
-                    ).format(latest_release)
-                elif is_pynsist() or running_in_mac_app():
-                    box.setStandardButtons(QMessageBox.Yes |
-                                           QMessageBox.No)
-                    content = _(
-                        "Would you like to automatically download and "
-                        "install it?<br><br>"
-                    )
-
-                msg = header + content + footer
+
+                msg = header
+                if os.name == "nt" or sys.platform == "darwin":
+                    # Always show download option for Windows and macOS
+                    box.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+                    msg += download_msg
```

Comment:
```suggestion
                    msg += download_msg
                    if not (is_pynsist() or running_in_mac_app()):
                        msg += not_installer_message
```

Add the appropriate message if not running in a standalone installer

Reply:
?

Reply:
See above—I renamed the variable here but not there.